browse/scripts/main.py

```diff
@@ -37,10 +37,9 @@
 f.gen_out("../../database/db.csv", "../alpha/z.md", "alphabet", "z")
 
 # topics
-# use the topics parser to get a list of available topics and generate all 
-# markdown files as per that list.
-
 tp = TopicsParser()
 topics = tp.gen_topics("../../database/db.csv")
+topics_file = open("../topics/01-topics-list.md", "w", encoding="UTF-8")
```

Comment:
The file name is 00-xx. Any specific reason for renaming it as 01-xx?

Reply:
anyway, I fixed at my end and also did some other corrections. Merged and live on the website.
Thanks a lot! 😃

Reply:
Sorry yea I thought I had swapped that back it was for testing so I could make sure I could see what I was actually writing versus what was written before. Thanks for changing it!